ToDoWebApi/Controller/UserController.cs

```diff
@@ -85,4 +89,22 @@ public IActionResult DeleteUser(int id, string email, string password)
         command.Handle();
         return Ok();
     }
+
+    [HttpPost("connect/token")]
+    public ActionResult<Token> CreateToken([FromBody] CreateTokenViewModel login)
+    {
+        CreateTokenCommand command = new CreateTokenCommand(_context, _mapper, _configuration);
+        command.Model = login;
+        var token = command.Handle();
+        return token;
+    }
+
+    [HttpGet("refreshToken")]
+    public ActionResult<Token> RefreshToken([FromQuery] string token)
+    {
+        RefreshTokenCommand command = new RefreshTokenCommand(_context, _configuration);
+        command.RefreshToken = token;
+        var resultToken = command.Handle();
+        return resultToken;
```

Comment:
bu iki command için validator kullanmaya ihtiyaç yok mu?

Reply:
Create Token için validator kullanıldı. Refresh zaten gerek duyulmadı.